### PR TITLE
[VYMCA-76] Categories enhancements for blogs and videos + Make event category multiple

### PR DESCRIPTION
**Pull request by hamrant**
### Steps for review:

- [ ] Login as admin
- [ ] Go to /admin/content?title=&type=vy_blog_post&status=All&langcode=All
- [ ] Edit any blog post
- [ ] Check that blogs has Category
- [ ] Go to application home page
- [ ] Press View All on videos
- [ ] Open any categoy and check that you can see videos here

Screenshot from 2020-09-15 12-12-36

- [ ] Go to application home page
- [ ] Press View All on blogs
- [ ] Check that you can see here only the blog category (Health)
- [ ] Open category and check that you can see blogs here

Screenshot from 2020-09-15 12-14-19

- [ ] Open Any blog and check that you can see the Category link under the title and lead to this category
- [ ] Edit any blog and add the second category
- [ ] Check that category displayed on the /virtual-ymca#/categories/blog
- [ ] Check that two categories displayed under the title on the blog page
- [ ] Go to /admin/content/events/series
- [ ] Edit any event
- [ ] Check that category field multiple and add the second category
- [ ] Open event page in the application
- [ ] Check that it displays several categories in details

Screenshot from 2020-09-15 12-21-19

- [ ] Edit virtual meeting and delete description
- [ ] Check that https://github.com/ymcatwincities/openy_gated_content/issues/32 fixed

### CODE REVIEW

File: js/gated-content/src/views/CategoryPage.vue [hunk 1]
```diff
@@ -3,19 +3,33 @@
...
+          <div
+            v-if="category.attributes.description"
+            v-html="category.attributes.description.processed"
+            class="mb-3"
+          ></div>
+        </div>
+      </div>
+      <div class="back-to-categories-wrapper">
+        <div class="gated-container">
+          <router-link :to="{ name: 'CategoryListing' }">← Back to all categories</router-link>
```
Comment (NightHunterSV) [inline js/gated-content/src/views/CategoryPage.vue:18]: Shouldn't we have a param `type` here?

Comment (hamrant) [reply js/gated-content/src/views/CategoryPage.vue:18]: it works fine without param

File: src/Controller/CategoriesController.php [hunk 2]
```diff
@@ -41,10 +41,11 @@ public static function create(ContainerInterface $container) {
   /**
    * Provides a list of categories uuid's that contains videos.
```
Comment (NightHunterSV) [inline src/Controller/CategoriesController.php:42]: Would be good to update the function description and param info

Comment (hamrant) [reply src/Controller/CategoriesController.php]: Fixed

**Review by NightHunterSV — APPROVED** [review]

File: js/gated-content/src/views/CategoryPage.vue (hunk 1 above)
Comment (hamrant) [reply js/gated-content/src/views/CategoryPage.vue:18]: @NightHunterSV I think it reused type from the current route (that also contains this param)

**Review by anpolimus — DISMISSED** [review]

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
Pr tested. Everything is fine.

**Pull request merged**